test/capwap_dummy_app.erl

```diff
@@ -0,0 +1,15 @@
+-module(capwap_dummy_app).
+
+-include_lib("kernel/include/logger.hrl").
+
+-behaviour(application).
+
+%% Application callbacks
+-export([start/2, stop/1]).
+
+start(StartType, StartArgs) ->
+    ?LOG(debug, "Starting with terms: ~p", [{StartType, StartArgs}]),
+    capwap_dummy_sup:start_link(StartArgs).
+
+stop(State) ->
+    ?LOG(debug, "Stopping: ~p", [State]).
```

Comment:
Please add new line in the end of module

Reply:
What do you mean exactly?

Reply:
If you will take a look on the 15th line and look a little lower you see a red circle indicating that there is `no newline end of file`

Reply:
I'll check if this is an encoding issue.

Reply:
This is related to specific settings of code editor

Reply:
Solved